.rubocop.yml

```diff
@@ -117,26 +117,18 @@ Rails/DynamicFindBy:
   Exclude:
     - "app/sample_manifest_excel/**/*"
     - "test/app/sample_manifest_excel/**/*"
-  Whitelist:
+  AllowedMethods:
     - find_by_sql
     - find_by_id_or_name
     - find_by_id_or_name!
     - find_by_barcode
-    - find_by_or_null
 # Method length limits make little sense when you are describing a table schema
 # or a view.
 Metrics/MethodLength:
   Exclude:
     - "db/migrate/*"
-    - "db/migrate_asset_refactor/*"
-Layout/FirstArrayElementIndentation:
-  EnforcedStyle: consistent
```

Comment:
Indentation is now under control of prettier. It configures the layout cops to be compatible, so lets not get in its way. (Even if we seem to be agreeing at the moment)

Reply:
Yep cool